### PR TITLE
Added more info about EWC

### PR DESCRIPTION
**Pull request by skremdwd**
I added the discussed information about the EWC.

As I have no experience with the used doc generator tool and I don't have any local Docker installation to use I will try to compile the docs locally with asiidoctor-pdf.

Or is this part already automated via a action in github?

### CODE REVIEW

File: source/all.adoc [hunk 1]
```diff
@@ -443,7 +443,16 @@ These tests should include but are not limited to:
 
 == Deployment and Operations
 
-All environments run in the EWC.
+All environments run in the EWC. The EWC uses Virtual Machine instances with the following currently possible configuration options:
+
+* Ressources
```
Comment (mortenwh) [inline source/all.adoc:448]:
```suggestion
* Resources
```

**Review by mortenwh — APPROVED** [review]
Great - thanks! Minor typo correction but then you can merge :)

File: source/all.adoc (hunk 1 above)
Comment (mortenwh) [reply source/all.adoc:448]: hm - or wasn't it ready? I now see it is still in draft mode... Maybe @myggen could take a look anyway...

**Comment by skremdwd** [comment]
From my side it is ready, just the generated content is missing. I marked it therefore as draft. I need a bit more time to look into it, as I can't find any package in the repository of my OS for asciidoctor(-pdf). If somebody can generate this quickier it's fine for me else I will try it again this afternoon.

**Comment by mortenwh** [comment]
@skremdwd - you can use vagrant or the compile-docs.sh script. With vagrant, just install vagrant and virtualbox, then run `vagrant up`.

**Comment by skremdwd** [comment]
I just generated the docs, I installed Ruby and used the gem installer, this was a bit easier.
Maybe squash this commit is the best merge option to not pollute the commit history.

**Pull request merged**